CHANGELOG.md

```diff
@@ -1,5 +1,152 @@
+  The flag to specify the key name for the CLI command `keys add` has been changed
+  from `-n` to `-k`. Restoring a key now takes a file containing the mnemonic as
+  input instead of directly taking the mnemonic
+- Deprecated `gas_adjustment` setting in favor of new `gas_multiplier` setting
+- Updated all CLI commands to take flags instead of positional arguments
+- Renamed `query packet unreceived-packets` to `query packet pending-sends`
+  and `query packet unreceived-acks` to `query packet pending-acks`
+- Added CLI command `keys balance` which outputs the balance of an account associated with a key
+- Added CLI command `query channel client` which outputs the channel's client state
+- Added CLI command `query transfer denom-trace` which outputs the base denomination and path of a given trace hash
+- Dropped the `raw` prefix from all the `tx raw` commands
+- Remove the four duplicate commands:
+  * `tx raw update-client`, which is the same as `update client`
+  * `tx raw upgrade-client`, which is the same as `upgrade client`
+  * `tx raw upgrade-clients`, which is the same as `upgrade clients`
+  * `tx raw create-client`, which is the same as `create client`
+- [A new section was added to guide][telemetry-guide] which describes how the new metrics
+  can be used to observe both the current state of the relayer and the networks it is connected to
+- Added many new metrics to the telemetry. The full list can be found in new the guide section linked above
+
+### Change to the versioning scheme
+
+As of v1.0.0-rc.0, the Hermes CLI is now versioned separately from
+the other crates in the project. As such, the top-level version
+designates the version of the Hermes CLI, but the other crates in
+the repository do not necessarily match this version. For example,
+the `ibc` and `ibc-relayer` crates are released under version 0.19.0
+for Hermes v1.0.0.
+
+The structure of this changelog has therefore changed as well,
+changes are now grouped first by crate and then by the type of change,
+eg. feature, bug fix, etc.
+
+### Full release notes
+
+The release notes below only contain the changes introduced since v1.0.0-rc.2.
+For the full list of changes since v0.15.0, please consult the sections below for
+v1.0.0-rc.2, v1.0.0-rc.1 and v1.0.0-rc.0.
+
+### General
+
+- Bumped crates to the following versions:
+  - `ibc-relayer-cli`: 1.0.0
+  - `ibc-proto`: 0.20.1
+  - `ibc`: 0.19.0
+  - `ibc-relayer`: 0.19.0
+  - `ibc-telemetry`: 0.19.0
+  - `ibc-relayer-rest`: 0.19.0
+
+### Hermes - [`ibc-relayer-cli`](relayer-cli) (v1.0.0)
+
+- Release version 1.0.0 of Hermes (`ibc-relayer-cli`)
+
+### IBC Proto - [`ibc-proto`](proto) (v0.20.0)
+
+- Release version 0.20.1 of `ibc-proto`
+
+### IBC Modules - [`ibc`](modules) (v0.19.0)
+
+- Release version 0.19.0 of `ibc`
+
+### BREAKING CHANGES
+
+- Remove `height` attribute from `IbcEvent` and its variants
+  ([#2542](https://github.com/informalsystems/ibc-rs/issues/2542))
+
+### BUG FIXES
```

Comment:
```suggestion
#### BUG FIXES
```

Reply:
Note that these tags are automatically generated I think. We might want to upstream your suggestions into `unclog` crate.